### PR TITLE
Fix Bug 1228015, reduced font size on eoy home promo

### PR DESCRIPTION
**Pull request by schalkneethling**
@alexgibson @flodolo r? Main locales tested are:
en-gb, en-ca, de, pt-br, de, es-es, it and nl

### CODE REVIEW

File: media/css/mozorg/home/home-promo.less [hunk 1]
```diff
@@ -1105,6 +1107,27 @@ html[lang|="en"] {
     }
 }
 
+html[lang="de"],
+html[lang="it"] {
```
Comment (flodolo) [inline media/css/mozorg/home/home-promo.less]: I'm not seeing any problem with Italian and the current font-size. Are you?

File: media/css/mozorg/home/home-promo.less [hunk 2]
```diff
@@ -1105,6 +1107,27 @@ html[lang|="en"] {
...
+    .promo-large-landscape.fundraiser {
+        h2 {
+            .font-size(24px);
+            line-height: 1.2;
```
Comment (flodolo) [inline media/css/mozorg/home/home-promo.less]: (not l10n) Do you need to repeat this?

**Comment by flodolo** [comment]
I don't think we need Italian, but we definitely need to fix 'nl' when the tile is vertical

schermata 2015-11-26 alle 16 06 18

Probably a good idea to include es-ES. The text is currently on 3 lines, and the arrow sits right over the edge.

File: media/css/mozorg/home/home-promo.less (hunk 1 above)
Comment (schalkneethling) [inline media/css/mozorg/home/home-promo.less]: It looked like the initial state had a really long second line so, thought it best to lessen the size. Perhaps not needed.

File: media/css/mozorg/home/home-promo.less (hunk 2 above)
Comment (schalkneethling) [inline media/css/mozorg/home/home-promo.less]: Ha! Could have sworn I removed it.

**Comment by alexgibson** [comment]
r+ :earth_asia:

**Pull request merged**